.github/workflows/ci.yml

```diff
@@ -121,7 +121,7 @@ jobs:
           build: 'false'
 
       - uses: './.github/actions/setup-woocommerce-monorepo'
-        if: ${{ github.ref_type != 'tag' }}
+        if: ${{ github.ref_type != 'tag' &&  matrix.testEnv.shouldCreate }}
```

Comment:
Nice! Do we have the means to skip the step for PHP unit tests?

Reply:
I'm not sure there is a case where we want it skipped for unit tests.

Reply:
I can evaluate it in a separate PR, but looking into build commands, WooCommerce build steps are around building assets. I assume PHPUnit will not need them to run tests.

Reply:
I'll draft a PR today to test my proposal - let's skip it for this PR to move faster.

Reply:
Drafted https://github.com/woocommerce/woocommerce/pull/49135, marking this thread as resolved.